src/CattyTests/Bricks/IfOnEdgeBounceBrickTests.m

```diff
@@ -1,295 +0,0 @@
-/**
```

Comment:
This deprecated file is not used anymore (converted to Swift) and has been removed from the file system.